src/commutator_rpnl.F

```diff
@@ -398,4 +402,509 @@ SUBROUTINE build_com_rpnl(matrix_rv, qs_kind_set, sab_orb, sap_ppnl, eps_ppnl)
 
    END SUBROUTINE build_com_rpnl
 
+! **************************************************************************************************
+!> \brief ...
+!> \param qs_kind_set ...
+!> \param sab_all ...
+!> \param sap_ppnl ...
+!> \param eps_ppnl ...
+!> \param matrix_rv ...
+!> \param matrix_rxrv ...
+!> \param matrix_rrv ...
+!> \param ref_point ...
+!> \param particle_set ...
+!> \param cell ...
+! **************************************************************************************************
+   SUBROUTINE build_com_mom_nl(qs_kind_set, sab_all, sap_ppnl, eps_ppnl, matrix_rv, matrix_rxrv, matrix_rrv, ref_point, &
+                               particle_set, cell)
+
+      TYPE(qs_kind_type), DIMENSION(:), POINTER          :: qs_kind_set
```

Comment:
`INTENT(...)` missing. I know you probably took some other routine as an example, but could you please add appropriate `INTENT(IN/OUT/INOUT)`?